asapdiscovery-alchemy/asapdiscovery/alchemy/utils.py

```diff
@@ -156,17 +153,8 @@ def action_network(
                 self._client.create_tasks(tf_sk, count=planned_network.n_repeats + 1)
             )
 
-        # set the network weight based on the found network weights that are currently
-        # running while making sure the weight doesn't fall outside 0.1-1 as 0.0 turns of compute.
-        if prioritize is True:
-            weight = np.clip(max(self.get_actioned_weights()) + 0.01, 0.1, 1.0)
-        elif prioritize is False:
-            weight = np.clip(min(self.get_actioned_weights()) - 0.01, 0.1, 1.0)
-        else:
-            weight = 0.5
-
         # now action the tasks to ensure they are picked up by compute.
-        actioned_tasks = self._client.action_tasks(tasks, network_key, weight=weight)
+        actioned_tasks = self._client.action_tasks(tasks, network_key, weight=0.5)
```

Comment:
Document default of 0.5 somewhere.

Reply:
we don't actually need to set this because the `alchemiscale` default is already 0.5 (documented on their side)